src/moonlink/src/storage/mooncake_table.rs

```diff
@@ -499,6 +499,9 @@ pub struct MooncakeTable {
 
     /// Table notifier, which is used to sent multiple types of event completion information.
     table_notify: Option<Sender<TableEvent>>,
+
+    // LSNs of pending flush operations.
+    pending_flush_lsns: BTreeSet<u64>,
```

Comment:
why do we put half of the flush logic in table handler, and another half in mooncake table?